### PR TITLE
Added percentage-change for all metrics API

### PR DESCRIPTION
**Pull request by pabragin**
Since we updated API recently https://dev.chartmogul.com/reference/retrieve-all-key-metrics we also need to add support for percentage-change in Ruby lib

### CODE REVIEW

File: lib/chartmogul/api/actions/all.rb [hunk 1]
```diff
@@ -11,6 +11,7 @@ def self.included(base)
         module ClassMethods
           def all(options = {})
             resp = handling_errors do
+              puts resource_path.apply_with_get_params(options)
```
Comment (SrikalaB) [inline lib/chartmogul/api/actions/all.rb:14]: Is this puts supposed to be here?

Comment (pabragin) [reply lib/chartmogul/api/actions/all.rb]: Oh, no, I don't even remember why I did this.

File: lib/chartmogul/summary_all.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module ChartMogul
+  class SummaryAll < ChartMogul::Object
```
Comment (SrikalaB) [inline lib/chartmogul/summary_all.rb:4]: Why are we calling it SummaryAll - Isn't the end point MetricsAll?

Comment (pabragin) [reply lib/chartmogul/summary_all.rb:4]: Endpoint - yes. But it is only represents Summary object for MetricsAll endpoint, similar to Summary for other endpoints: https://github.com/chartmogul/chartmogul-ruby/blob/3291c3a21b7c28a9d27a93711728822b3b90108a/lib/chartmogul/summary.rb

**Review by SrikalaB — APPROVED** [review]

**Pull request merged**